### PR TITLE
#163426619 Integrate Coveralls code coverage

### PR DESCRIPTION
**Pull request by ODINAKACHUKWU**
#### What does this PR do?
Have coveralls to determine the level of coverage on the code base

#### Description of Task to be completed?
- install coveralls and nyc to generate test coverage reports
- add coverage ReadMe badge

#### What are the relevant pivotal tracker stories?
#163426619

### CODE REVIEW

**Review by eokwukwe — COMMENTED** [review]
Looks good so far

**Review by Ndifreke — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (obasajujoshua31) [inline package.json:7]: This test scripts has not been edited, I guess Benny talked about it yesterday during our daily standups

**Review by obasajujoshua31 — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Looks good, but please fix the test script.

**Review by mrkingz — COMMENTED** [review]
Looks good. Thumbs up!

**Pull request closed without merging**